SabkTrial/Scenes/News Screen/NewsScreenCells/ArticlesCollectionViewCell.swift

```diff
@@ -10,14 +10,27 @@ import UIKit
 
 class ArticlesCollectionViewCell: UICollectionViewCell {
 
-    @IBOutlet weak var aothorName: UILabel!
-    @IBOutlet weak var authorImage: UIImageView!
-    @IBOutlet weak var articleTitle: UILabel!
+    @IBOutlet private weak var authorName: UILabel!
+    @IBOutlet private weak var authorImage: UIImageView!
+    @IBOutlet private weak var articleTitle: UILabel!
     override func awakeFromNib() {
         super.awakeFromNib()
         // Initialization code
         authorImage.layer.masksToBounds = true
         authorImage.layer.cornerRadius = authorImage.bounds.width / 2
     }
-
+    
+    func setAuthorName(text: String) {
+        authorName.text = text
+    }
+    
+    func setArticleTitle(text: String) {
+        articleTitle.text = text
+    }
+    
+    func setAuthorImage(imageURL: String) {
+      authorImage.sd_setImage(
+            with: URL(string: imageURL),
+            placeholderImage: #imageLiteral(resourceName: "img_placeholder"))
+    }
 }
```

Comment:
Just one function with all of them will be better

Reply:
done